index.html

```diff
@@ -564,42 +564,46 @@ <h2>
         This example acquires a screen wake lock and releases it after a while:
       </p>
       <pre class="example">
-        navigator.getWakeLock("screen").then(function(wakeLock) {
-          var request = wakeLock.createRequest();
-          setTimeout(function() {
-            request.cancel();
-          }, 1000);
-        });
+        async function tryKeepScreenAlive(minutes) {
+          const lock = await navigator.getWakeLock("screen");
+          const request = lock.createRequest();
+
+          setTimeout(() => request.cancel(), minutes * 60 * 1000);
+        }
```

Comment:
To make this example complete, add:
```
tryKeepScreenAlive(10);
```

Reply:
Done also tested all examples and made sure they work